project/Build.scala

```diff
@@ -2176,10 +2174,21 @@ object Build {
       packArchiveName := (dist / packArchiveName).value + "-x86_64-pc-win32",
       republishBinOverrides += (dist / baseDirectory).value / "bin-native-overrides",
       republishFetchCoursier := (dist / republishFetchCoursier).value,
-      republishExtraProps += ("cli_version" -> scalaCliLauncherVersion),
-      mappings += (republishRepo.value / "EXTRA_PROPERTIES" -> "EXTRA_PROPERTIES"),
       republishLaunchers +=
-        ("scala-cli.exe" -> s"zip+https://github.com/VirtusLab/scala-cli/releases/download/v$scalaCliLauncherVersionWindows/scala-cli-x86_64-pc-win32.zip!/scala-cli.exe")
+        ("scala-cli.exe" -> s"zip+https://github.com/VirtusLab/scala-cli/releases/download/v$scalaCliLauncherVersion/scala-cli-x86_64-pc-win32.zip!/scala-cli.exe")
+    )
+    .settings(
+      Windows / name := "scala",
+      Windows / mappings   := (Universal / mappings).value,
+      Windows / packageBin := (Windows / packageBin).dependsOn(republish).value,
+      Windows / wixFiles   := (Windows / wixFiles).dependsOn(republish).value,
+      // Additional information: https://wixtoolset.org/docs/schema/wxs/package/
+      maintainer := "The Scala Programming Language",                             // The displayed maintainer of the package
+      packageSummary := s"Scala $dottyVersion",                                   // The displayed name of the package
+      packageDescription := """Installer for the Scala Programming Language""",   // The displayed description of the package
+      wixProductId := "*",                                                        // Unique ID for each generated MSI; will change for each generated msi
+      wixProductUpgradeId := "3E5A1A82-CA67-4353-94FE-5BDD400AF66B",              // Unique ID to identify the package; used to manage the upgrades
+      wixProductLicense := Some(dist.base / "LICENSE.rtf")                        // Link to the LICENSE to show during the installation (keep in sync with ../LICENSE)
```

Comment:
This should not be backported !!

Reply:
Original commit: https://github.com/scala/scala3/pull/21234/commits/0f0b703a0eb82c693a2f2edfe0932232ae44b10f

Reply:
Correct, thank you. I've reverted these too quick changes done when resolving conflicts.